File: test/tw/rmstudio/mt/java/course/mcu/ds/ex1/test/TestShellSort.java
package tw.rmstudio.mt.java.course.mcu.ds.ex1.test;

import tw.rmstudio.mt.java.course.mcu.ds.ex1.sorting.ShellSort;

public class TestShellSort extends TestSortingAlgorithm {
    @Override
    int[] sort(int[] randomNumbers) {
        return new ShellSort().sortResult(randomNumbers).getSortedNumbers();
    }
}

File: test/tw/rmstudio/mt/java/course/mcu/ds/ex1/test/TestShellSort.kt
package tw.rmstudio.mt.java.course.mcu.ds.ex1.test

import tw.rmstudio.mt.java.course.mcu.ds.ex1.sorting.ShellSort

class TestShellSort : TestSortingAlgorithm() {
    override fun sort(randomNumbers: IntArray) = ShellSort().sortResult(randomNumbers).sortedNumbers
}
